apps/ffdweb-site/src/app/digest/[slug]/components/AuthorsBio.tsx

```diff
@@ -0,0 +1,15 @@
+import { formatAuthors } from '@filecoin-foundation/utils/formatAuthors'
+import type { AuthorExtended } from '@filecoin-foundation/utils/types/authorTypes'
+
+type AuthorBioProps = {
+  authors: Array<AuthorExtended>
+}
+
+export function AuthorsBio({ authors }: AuthorBioProps) {
+  return (
+    <div className="flex flex-col gap-2 sm:w-2/3">
+      <h2 className="text-base font-bold">{formatAuthors({ authors })}</h2>
```

Comment:
Not sure an h2 is appropriate for this? Might not need to be a heading?

Reply:
For me it made semantic sense, but happy to go with paragraph